src/js/yeldaWebchatInjector.js

```diff
@@ -534,10 +493,14 @@ class YeldaChat {
    * Delete old webchat element and create new webchat
    * @param {Object} data { data.assistantUrl, data.chatPath }
    * @param {Element} container webchat container
+   * @returns {Promise}
   */
   resetChat (data) {
-    this.unLoadChat()
-    this.setupChat(data)
+    return new Promise(async resolve => {
+      this.unLoadChat()
+      await this.setupChat(data)
+      resolve()
+    })
   }
```

Comment:
we have a consistency issue : why is suddently resetChat async (i can see it's awaited in the test) and not init ?
once again are we sure it's well supported on all browser ?